app/models/password_history.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: password_histories
+#
+#  id                     :integer          not null, primary key
+#  user_id                :integer           default(''), not null
+#  encrypted_password     :string           default(''), not null
+
+class PasswordHistory < ApplicationRecord
+  belongs_to :user
+
+  validates :encrypted_password, presence: true, uniqueness: true
```

Comment:
There should be a basic unit test for this new model

Reply:
@yasirazgar I don't think you committed the `password_histories_spec`

Reply:
Oops, my bad committed now.